src/vs/base/browser/ui/grid/gridview.ts

```diff
@@ -464,6 +464,7 @@ class BranchNode implements ISplitView<ILayoutContext>, IDisposable {
 		this.absoluteOrthogonalOffset = ctx.absoluteOrthogonalOffset;
 		this.absoluteOrthogonalSize = ctx.absoluteOrthogonalSize;
 
+		// TODO: Some thing here seem cause these offsets to be wrong
```

Comment:
@sbatten @joaomoreno I will probably need your help with this change. From my investigation, I believe the offsets being passed down are incorrect (or I am not understanding how to use them).

With the sidebar collapsed and a notebook in the first column, `EditorGroupView` is passed the following offsets:

```
top: 28
left: 96
```

The value of `left` (96) seems to be double what it should be since the activity bar is only 48px wide. I've confirmed that the value is also doubled when the sidebar is showing

The value of `top` (28)  seems correct since it only includes the height of the title bar. For editors, we then add in the height of the the breadcrumbs and tabor

Reply:
Cool that you're addressing https://github.com/microsoft/vscode/issues/129120#issuecomment-950984363. I've updated the PR description to create a link to that comment..

The fix in https://github.com/microsoft/vscode/issues/129120 had a typo. This should fix it: https://github.com/microsoft/vscode/pull/164327